modules/jupyter/main.tf

```diff
@@ -17,6 +17,7 @@ data "google_project" "project" {
 }
 
 locals {
+  cloudsql_instance_connection_name = var.cloudsql_instance_name != "" ? format("%s:%s:%s", var.project_id, var.db_region, var.cloudsql_instance_name) : ""
```

Comment:
could you please try to run tf apply from /applications/jupyter to ensure that the secrets mounting isn't breaking just the jupyter installation? it should be fairly quick to test

Reply:
Good call. Fixed an error.